projects/driver_controls/src/gpio_expander.c

```diff
@@ -5,6 +5,15 @@
 #include "i2c.h"
 #include "mcp23008.h"
 
+static void prv_poll_timeout(SoftTimerID timer_id, void *context) {
+  GpioExpanderStorage *expander = context;
+
+  // Trigger an interrupt to force an update
+  gpio_it_trigger_interrupt(&expander->int_pin);
+
```

Comment:
Optional nit: unnecessary newline. Generally I prefer keeping things close together in short functions. For longer functions newlines are fine to break up logical blocks. This is a personal thing though so up to you.